File: GPL_Problems/LeetCode_Problem_3300_Solution_1.java
class Solution {
    public int minElement(int[] nums) {

        long sum = 0;

        for (int i = 0; i < nums.length; i++) {

            while (nums[i] != 0) {
                sum += nums[i] % 10;
                nums[i] = nums[i] / 10;
            }

            nums[i] = (int) sum;
            sum = 0;
        }

        long rtrn_min = nums[0];

        for (int i = 1; i < nums.length; i++) {
            if (nums[i] < rtrn_min) {
                rtrn_min = nums[i];
            }
        }

        return (int) rtrn_min;
    }
}

File: GPL_Problems/LeetCode_Problem_3300_Solution_1.cpp
class Solution {
public:
    int minElement(vector<int>& nums) {
        long long sum = 0;

        for(int i = 0; i < nums.size(); i++){
            while(nums[i] != 0){
                sum += nums[i] % 10;
                nums[i] = nums[i] / 10;
            }
            nums[i] = sum;
            sum = 0;
        }

        long long rtrn_min = nums[0];

        for(int i = 1; i < nums.size(); i++){
            if(nums[i] < rtrn_min){
                rtrn_min = nums[i];
            }
        }

        return rtrn_min;
    }
};